OsmAnd/src/net/osmand/plus/track/helpers/GPXDatabase.java

```diff
@@ -225,6 +225,14 @@ public class GPXDatabase {
 			GPX_COL_MAX_FILTER_ALTITUDE + " = ?, " +
 			GPX_COL_MAX_FILTER_HDOP + " = ? ";
 
+	private static final String GPX_NEAREST_CITY_LIST = "SELECT DISTINCT " +
+			GPX_COL_NEAREST_CITY_NAME + " FROM " + GPX_TABLE_NAME +
+			" WHERE " + GPX_COL_NEAREST_CITY_NAME + " NOT NULL";
+
+	private static final String GPX_MIN_DATE = "SELECT " +
+			"MAX(" + GPX_COL_FILE_LAST_MODIFIED_TIME + ") " +
```

Comment:
GPX_MIN_DATE but we search for MAX fileLastModifiedTime, probably it should be renamed?